src/NodeApi/JSReference.cs

```diff
@@ -165,14 +176,36 @@ T GetValueAndRunAction()
 
     public bool IsDisposed { get; private set; }
 
-    private void ThrowIfDisposed()
+    private void CheckDisposed()
```

Comment:
The previous name `ThrowIdDisposed` was much better. It was easy to read and understand. The new name looks "smoother" but difficult to understand without looking at the implementation. It says the completely opposite: it says that it checks that the object is already disposed, but in reality, it checks that it is not disposed.
We should either keep the previous name or rename it to `CheckNotDisposed`.
I would rather keep the existing name unless the .Net SDK has already predefined pattern in that context.

Reply:
I had changed this to be consistent with the new thread access check method, but didn't think carefully about it. I renamed them both to `ThrowIf...`.